vc_vidyo/indico_vc_vidyo/api/client.py

```diff
@@ -79,9 +79,28 @@ def create_room_object(self, **kwargs):
     def find_room(self, extension):
         filter_ = self.client.factory.create('Filter')
         filter_.query = extension
+        filter_.limit = 40
         filter_.dir = 'DESC'
 
-        return self.soap.getRooms(filter_).room
+        target_room = None
+        counter = 0
+        from indico_vc_vidyo.plugin import VidyoPlugin
```

Comment:
I'm quite sure it'd cause a circular import here - but I don't think it's needed here at all, the log messages are not really useful except when testing the code locally